data/json/furniture_and_terrain/furniture-triffid.json

```
@@ -0,0 +1,40 @@
+[
+  {
+    "type": "furniture",
+    "id": "f_glowingbulb",
+    "name": "illuminating plant bulb",
+    "description": "A small plant grows in the roots here, its green onion-like bulb illuminating the area around it.",
```

```suggestion
    "description": "A small plant grows among the roots here, its green onion-like bulb illuminating the area around it.",
```
I'm not quite sure about the intended visuals. Is it growing out of the roots, or among them?

---

Among them yeah